src/styles/clock.scss

```diff
@@ -0,0 +1,87 @@
+.content {
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

Comment:
use mixins for styles that you used more than  2 times

Reply:
corrected.